cookbooks/simple_subduction_2d_cartesian/doc/README.md

```diff
@@ -0,0 +1,222 @@
+creating a grid file so you can check the construction of your model as you add in 
+additional features by visualizing the model output using Paraview (or similar). 
+A grid file has a specific structure defining the geometry, extent of the model region 
+to visualize and the grid spacing. Choose a grid spacing that is sufficient to check the 
+model is defined properly. For a 2D model only the information for the 2D grid of points 
+needs to be indicated in the grid file. For this example, the full model domain is 
+8000 km long and 1600 km deep.  The grid file for this example is:
+
+:::{literalinclude} ../simple_subduction_2d_cartesian.grid
+:::
+
+## Root model information
+
+In the root model section we need to indicate the geometry (Cartesian). Even though we 
+only want a 2D model, World Builder starts from a 3-D space. Therefore, we also need to 
+define a 2-D cross section through the 3D World Builder space. The `cross section` 
+parameter defines the start and end locations for the cross section at the surface of 
+the model (z = 0). We choose to put the cross section at y = 0, and give the end 
+points of the cross section at x = 0 and x = 8000e3 m. The other physical parameters 
+are defined as reference values, which will be used by the feature temperature models 
+if the values are not entered separately.
+
+```{literalinclude} ../simple_subduction_2d_cartesian.wb
+:language: json
+:lineno-start: 1
+:lines: 1-6
+```
+ 
+### Feature Geometry, temperature and composition 
+
+Because the World Builder defines features in 3D space, even though we want to define 
+a 2D model, we still need to indicate some lateral extent for features in the y-direction. 
+Any value can be chosen for this width, so we will center the cross section at y = 0, and 
+let the y-direction extend from -100e3 m to 100e3 m.
+
+#### Mantle Layer
+We start by defining the mantle layer to encompass all of the model domain. 
+We don't need to define a thickness for the mantle, but we need to define its lateral 
+extent. This is done by assigning the x, y coordinates of the vertices of a polygon 
+given in clockwise order. In this example, the polygon is a rectangle, so there are 4 
+pairs of coordinates provide.  Because this is the first feature in the parameter file, 
+features that follow and overlap with this layer will replace (default) the existing value. 
+
+The reason we define a mantle layer is so we can set the background temperature to a u
+uniform value of 1573 K. The same value will be used as the bottom temperature in the 
+plate models. A constant temperature mantle is a simplification, but is some times useful 
+for testing ideas. A large model domain size is chosen to reduce the effects of the side 
+boundaries, However, an even wider model is likely needed with a depth of only 1600 km. 
+Alternatively, the box depth can be increased to 3000 km.  
+
+```{literalinclude} ../simple_subduction_2d_cartesian.wb
```

Comment:
The guide for documentation suggests posting several tabs with the whole file, etc .. Do you want that here?

Reply:
I will add the second tab for the full worldbuilder slab.